src/components/about/About.jsx

```diff
@@ -1,14 +1,17 @@
 export default function About() {
   return (
     <section className="">
-      <h1>About</h1>
-      <div className="container grid grid-cols-2 p-12 gap-8">
-        <div className="">
-          <img alt="teamwork logo" src="/illustrations/TeamWork.svg" />
+      {/* <div className="container grid grid-cols-2 p-12 gap-8">
+       */}
+       <div className="" style={{display:"grid",gridTemplateColumns: "repeat(auto-fit, minmax(400px, 1fr))"}}>
+        <div className="Image" style={{maxWidth: "100%", display:"block", margin:"auto"}}>
+          <img alt="teamwork logo" width='500px' src="/illustrations/TeamWork.svg"
```

Comment:
Why is the width attribute in image is a static value?

Reply:
width of the image should be same for both mobile and desktop!?